### PR TITLE
Added RedisGears 1.2.6 release notes

### PR DESCRIPTION
**Pull request by thomascaudron**
Added RedisGears 1.2.6 release notes, sourced from the [RedisGears repo release notes](https://github.com/RedisGears/RedisGears/releases/tag/v1.2.6).

### CODE REVIEW

File: content/modules/redisgears/release-notes/redisgears-1.2-release-notes.md [hunk 1]
```diff
@@ -14,11 +14,26 @@ aliases: /modules/redisgears/release-notes/redisgears-1.2-release-notes/
...
 
-RedisGears v1.2.5 requires:
+RedisGears v1.2.6 requires:
 
 - Minimum Redis compatibility version (database): 6.0.0
 - Minimum Redis Enterprise Software version (cluster): 6.0.12
 
+## 1.2.6 (March 2023)
+
+This is a maintenance release for RedisGears 1.2
```
Comment (nermiller) [inline content/modules/redisgears/release-notes/redisgears-1.2-release-notes.md:24]:
```suggestion
This is a maintenance release for RedisGears 1.2.
```

File: content/modules/redisgears/release-notes/redisgears-1.2-release-notes.md [hunk 2]
```diff
@@ -14,11 +14,26 @@ aliases: /modules/redisgears/release-notes/redisgears-1.2-release-notes/
...
+
+Update urgency: `LOW` : No need to upgrade unless there are new features or fixes.
```
Comment (nermiller) [inline content/modules/redisgears/release-notes/redisgears-1.2-release-notes.md:26]:
```suggestion
Update urgency: `LOW` - No need to upgrade unless there are new features or fixes.
```

File: content/modules/redisgears/release-notes/redisgears-1.2-release-notes.md [hunk 3]
```diff
@@ -14,11 +14,26 @@ aliases: /modules/redisgears/release-notes/redisgears-1.2-release-notes/
...
+
+### Details
+
+**Features:**
+- [#832](https://github.com/RedisGears/RedisGears/pull/832), [#844](https://github.com/RedisGears/RedisGears/pull/844) Added IPV6 support
+- [#841](https://github.com/RedisGears/RedisGears/pull/841) Generate artefacts with dependencies included
```
Comment (nermiller) [inline content/modules/redisgears/release-notes/redisgears-1.2-release-notes.md:32]:
```suggestion
- [#841](https://github.com/RedisGears/RedisGears/pull/841) Generate artifacts with dependencies included
```

File: content/modules/redisgears/release-notes/redisgears-1.2-release-notes.md [hunk 4]
```diff
@@ -14,11 +14,26 @@ aliases: /modules/redisgears/release-notes/redisgears-1.2-release-notes/
...
+
+**Bug fixes:**
+- [#810](https://github.com/RedisGears/RedisGears/pull/810) Fix invalid memory access when checking if a trigger is already register
```
Comment (nermiller) [inline content/modules/redisgears/release-notes/redisgears-1.2-release-notes.md:35]:
```suggestion
- [#810](https://github.com/RedisGears/RedisGears/pull/810) Fix invalid memory access when checking if a trigger is already registered
```

**Review by nermiller — APPROVED** [review]
some minor edits, Thomas

**Review by rrelledge — APPROVED** [review]

**Pull request merged**